SetupDataPkg/Tools/KnobService.py

```diff
@@ -184,15 +184,20 @@ def write_uefi_getter_implementations(efi_type, out, schema):
         if knob.help != "":
             out.write("// {}".format(knob.help) + get_line_ending(efi_type))
 
-        out.write("// Get the current value of the {} knob".format(knob.name) + get_line_ending(efi_type))
-        out.write("EFI_STATUS {}{} (".format(

```

Comment:
Can we add to the docs to explain why we have this backbending we are doing? And this is a new part of the system integration that people will have to implement.

Reply:
Good call, updated in the documentation.